src/components/course-review-card.js

```diff
@@ -5,15 +5,16 @@ import { Card } from 'semantic-ui-react';
 import { Rating } from 'semantic-ui-react';
 
 const CourseReviewCard = (props) => {
-  const { code } = props;
+  const { code, desc } = props;
 
   return (
-    <div style={{ display: 'block' }}>
+    <div className='my-card'>
```

Comment:
`course-review-card`, matching the component name `CourseReviewCard` would be more appropriate